Makefile.toml

```diff
@@ -284,3 +284,30 @@ run_task = "copy-lib"
 
 [tasks.post-android]
 dependencies = ["ffigen"]
+
+# TODO: non linux desktops
+# TODO: release build
+[tasks.desktop]
+description = "Build rust SDK for desktop"
+dependencies = [
+    "ffigen",
+    "desktop-build",
+    "copy-desktop-lib",
+]
+
+[tasks.desktop-build]
+description = "Build for desktop"
+command = "cargo"
+args = [
+    "build",
+    "-p",
+    "effektio"
+]
+
+[tasks.copy-desktop-lib]
+description = "Copy desktop library to plugin"
```

Comment:
```suggestion
[tasks.linux]
description = "Build rust SDK for linux desktop"
dependencies = [
    "ffigen",
    "linux-build",
    "copy-desktop-lib",
]

[tasks.linux-build]
description = "Build for linux desktop"
command = "cargo"
args = [
    "build",
    "-p",
    "effektio"
]

[tasks.copy-linux-lib]
description = "Copy linux desktop library to plugin"
```

Reply:
I'll later update this task to work on mac and windows as well.

Reply:
`DynamicLibrary.open` hangs on macos (unstable support). I've added support for linux and windows.